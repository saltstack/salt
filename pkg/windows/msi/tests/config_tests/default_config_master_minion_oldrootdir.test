properties START_MINION="" MASTER=cli.master.oldrootdir MINION_ID=cli.minion.oldrootdir INSTALLDIR=c:\salt ROOTDIR=c:\salt
dormant
