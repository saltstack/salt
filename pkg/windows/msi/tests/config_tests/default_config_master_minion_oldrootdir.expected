# This is the template from the msi store line 1/6
#master: salt
master: cli.master.oldrootdir
# This is the template from the msi store line 2/6
#id:
id: cli.minion.oldrootdir
# This is the template from the msi store line 3/6
# This is the template from the msi store line 4/6
# This is the template from the msi store line 5/6
# This is the template from the msi store line 6/6
